src/app/app.test.ts

```diff
@@ -25,6 +25,7 @@ export const config: IAppConfig = {
   oauthClientSecret: 'secret',
   cloudFoundryAPI: 'https://example.com/api',
   uaaAPI: 'https://example.com/uaa',
```

Comment:
Can we rename all instances of `uaaAPI` to `tokenAPI` so that it's consistent with the naming from `/v2/info` and easier to differentiate between the two endpoints (because we've already been confused by the `redirect_uri` problem)?